### DISCUSSION TITLE
看一看我国机关事业单位的养老标准 | /project/red-and-black/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#641] [Announcements]
https://yuanfan.rbind.io/project/red-and-black/

**Comment by botsp** [comment]
这算是另一种八旗吧，刻意地赎买一小部分人，就能极大地拉拢支持者。前些年台湾的军公教年金改革，也被指称是有计划地报复、整肃军公教群体，有口号是"不公不义", "社会的敌人"。

**Reply by earfanfan** [reply]
我想了想的，其实不能简单地说成是“刻意地赎买”，主要是因为我们国家的党政制度里面仍然有很大一部分“人治”的因素，也就是说其实存在明面和暗面两种同时存在的面，是一种人事与制度互相推动的东西，不过这点我还说不清楚。但从财政和行政的制度上来讲，上级将治理任务和指标一层层委托给下级政府，下级政府作为承包方只要完成了任务，在任务目标以外的领域有很大的自主发挥空间，结合本文来说就是，如果县政府已经征缴了达到任务目标的税收上解，剩下的如何分配的权利就在县政府手上，所以文中县与县之间的机关养老待遇也是有很大差异，有的县每年给每个退休干部发个十几万，有的发几万，至于其中多少是基金收入、多少是财政补助，不得而知。

历史上，每次农民起义以后，起义军领袖进城以后都会对有功的大肆分封奖赏，这样推翻一个国家是推翻了原先的阶级势力，重新建立一个国家又是建立了新的阶级势力，最初得势者必然会得到更多的特殊待遇，而最初的领导者也需要用那些更少的得势者来执行行政和管理的任务，拉拢也是必然的。这个军公教年金改革我稍微搜了搜，其实我也不懂这种改革应该在什么时机才必然要发生。感觉全社会存在许多不同的利益共同体，好像没有哪种改革能让所有人满意，或者说不会受到哪个群体的阻碍而顺利推行的。

**Reply by botsp** [reply]
部分赞同，一个社会集体总是要有相应的中产阶层的，军公教是传统上有知识、有技能的群体，值得相对高的收入或者养老待遇，只是在某些地区这种差异就会很刺眼。

比如台湾年金改革喊了很多年了，但最终在民进党的第二任总统蔡政府内解决，原因有很多，确是钱不够。可能最直接的原因就是，军公教是国民党的铁票仓，军公教长期受益于国民党在台湾的长期执政，算作是台湾版的体制内吧。彼此利益冲突，最终这个冲突以选票的形式体现，毕竟一人一票的情况下军公教还是社会少数。

我觉得这揭示一种可能，肯定不是说要"刻意霸凌"军公教群体，把钱给军公教总比发给八旗子弟、无赖流氓要好。但真正的社会共识会倾向于缩减退休年金的差距，也许一时无法解决，但总会被纠偏的。

**Reply by earfanfan** [reply]
>部分赞同，一个社会集体总是要有相应的中产阶层的，军公教是传统上有知识、有技能的群体，值得相对高的收入或者养老待遇，只是在某些地区这种差异就会很刺眼。

其实我前面写的并不是想要表达这样的观点，只是觉得在利益驱动下，我们生活的这块土地上总是会出现这样的问题。制度上太追求集体，集体中的个体就总会被“集体”这个概念绑架，因为集体要发展就要提高效率，各人禀赋不同，就不得不因为集体的目标是追求某一种效率而划分阶级，划分了阶级当然就会划分出人的价值等级了，最终就总会演化出巨大的价值差距。一个利益分化严重的集体……如果说我的观点的话，我觉得，额，这些差异刺眼不刺眼可能和信息公开的程度有关的，我的个人感觉是很多人更关注自己的生活，平时不怎么主动去看数据上的差异，对社会上的不平等的感触是很模糊的，只知道贫富差距大，也许也知道差距大到离谱，但不知道是多么广泛而具体存在的离谱。

>但真正的社会共识会倾向于缩减退休年金的差距，也许一时无法解决，但总会被纠偏的。

我也是这么想的，只是不知道纠偏真正能实现的时候是哪一天了。

对了，给你安利一篇文章[中国教育改革为什么会这么难](https://www.aisixiang.com/data/88347.html)，也许你会有兴趣看看。说实话 ，我觉得这篇文章切入的那几方面拿去分析社会上其他问题也是可以的。

**Reply by botsp** [reply]
这样的分析很有意义，把直观的感受具体化了，键盘政治也需要理论数据的支撑~ヾ(≧▽≦*)o
如果我们也有选票的话，应该很能拓展统计学的应用，问卷设计、抽样调查、数据报告和可视化(plus+两年一小次、四年一大次，县市省全国一片沸腾)，就业岗位还能再释放一波 :smile:

**Comment by earfanfan** [comment]
今天看到一则新闻，人大常委执法检查组出了一份报告，人大官网有一则[我国社会保险工作取得历史性成就](http://www.npc.gov.cn/npc/c2/c30834/202411/t20241107_440659.html)，其中有一段文字是这样写的。

>与此同时，我国社会保险保障水平也在不断提升。据统计，2012年至2023年，企业退休人员月人均养老金由1686元提高到3162元，城乡居民月人均养老金由82元提高到214元，失业保险金由707元提高到1814元，工伤保险月人均伤残津贴由1864元提高到4051元。

如果把2012、2022、2023年的人力资源与社会保障统计公报里与城镇职工基本养老保险的一些指标摘出来，只得到下面的数据。其中2022年城镇职工中企业的在职缴费和离退休人数数据源于国家统计局。

|年份|类别|参保人数（万人）|缴费人数（万人）|离退休人数（万人）|基金收入（亿元）|基金支出（亿元）|月人均养老金（元）|
|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
|2023|城镇职工基本养老保险|52121|37925|14196|70506|63757||
|2023|城镇职工-企业|46044|||||3162|
|2023|城镇职工-机关事业单位|||||||
|2022|城镇职工基本养老保险|50355|36711|13644|63323.8|59034.7||
|2022|城镇职工-企业||32817.5|11530.9||||
|2022|城镇职工-机关事业单位|||||||
|2012|城镇职工基本养老保险|30427|22981|7446|20001|15562||
|2012|城镇职工-企业|28272||6804.6|||1686|
|2012|城镇职工-机关事业单位|||||||

接下来，我要做一些非常粗糙的推算，主要目的是推算机关事业单位的月人均养老金上涨了多少。

其一，根据“2012年至2023年，企业退休人员月人均养老金由1686元提高到3162元”这句话，假设这11年来企业范围内的月人均养老金是匀速上涨，那么2022年的数值就是3028。

其二，缴纳城镇职工养老保险的除了企业、机关事业单位，应当还有一些灵活就业人员，但灵活就业人员的数据从未公布，因此本次推算假定城镇职工养老保险的数据中减去企业得到的就是机关事业单位的数据。

其三，这些指标之间应当有一些换算公式，参保人数=缴费人数+离退休人数，月人均养老金=(基金支出/离退休人数)/12。

那么接下来会推算出来这样的结果。

|年份|类别|参保人数（万人）|缴费人数（万人）|离退休人数（万人）|基金收入（亿元）|基金支出（亿元）|月人均养老金（元）|
|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
|2023|城镇职工基本养老保险|52121|37925|14196|70506|63757|3742.7|
|2023|城镇职工-企业|46044|||||3162|
|2023|城镇职工-机关事业单位|6077||||||
|2022|城镇职工基本养老保险|50355|36711|13644|63323.8|59034.7|3605.7|
|2022|城镇职工-企业|44348.4|32817.5|11530.9||41898.7|3028|
|2022|城镇职工-机关事业单位|6006.6|3893.5|2113.1||17136|6757.8|
|2012|城镇职工基本养老保险|30427|22981|7446|20001|15562|1741.7|
|2012|城镇职工-企业|28272|21467.4|6804.6||13767.1|1686|
|2012|城镇职工-机关事业单位|2155|1513.6|641.4||1794.9|2332|

根据上表推算的结果可以得到这样的一个结论：从2012年到2022年，机关事业单位的月人均养老金从2332元上涨到了6757.8元，企业的月人均养老金从1686元上涨到了3028元。在2012年，机关事业单位的月人均养老金是企业的1.38倍，而到2022年扩大到了2.23倍。
